Doc/c-api/init_config.rst

```diff
@@ -1143,6 +1144,20 @@ PyConfig
 
       Default: ``NULL``.
 
+   .. c:member:: wchar_t* run_presite
```

Comment:
Maybe rename it to just "presite".

Reply:
It's symmetrical to the other run_* PyConfig attributes that run things so I decided to keep the name.